core/sql/executor/ExExeUtilCli.cpp

```diff
@@ -1048,6 +1052,77 @@ Lng32 ExeCliInterface::executeImmediateExec(const char * stmtStr,
   return ((cliRetcode != -1) ? cliRetcode : retcode);
 }
 
+
+Lng32 ExeCliInterface::executeImmediateExecNoDealloc(const char * stmtStr,
+					   char * outputBuf,
+					   Lng32 * outputBufLen,
+					   NABoolean nullTerminate,
+					   Int64 * rowsAffected 
+					   )
+{
+  Lng32 retcode = 0;
+
+  retcode = exec();
+  if (retcode < 0)
+    {
+      deallocStuff(module_, stmt_, sql_src_, input_desc_, output_desc_);
+      return retcode;
+    }
+
+  retcode = fetch();
+  if (retcode < 0)
+    {
+      deallocStuff(module_, stmt_, sql_src_, input_desc_, output_desc_);
+      return retcode;
+    }
+
+  if ((outputBuf) &&
+      (outputBufLen))
+    {
+      *outputBufLen = 0;
+      if (retcode != 100)
+	{
+	  char * ptr;
+	  Lng32 len;
+	  getPtrAndLen(1, ptr, len);
+
+	  str_cpy_all(outputBuf, ptr, len);
```

Comment:
executeImmediateExeNoDealloc is not needed. Had added it when I encountered a bug with cursor access and then forgot to remove . The whole method has been removed.